Project.toml

```diff
@@ -0,0 +1,10 @@
+name = "JLD2"
+uuid = "30b3fee2-9826-11e8-2ba3-a7612145d6ce"
```

Comment:
Should use https://github.com/JuliaRegistries/General/blob/master/J/JLD2/Package.toml#L2